config/application.rb

```diff
@@ -37,6 +37,9 @@ class Application < Rails::Application
 
     config.autoload_paths << Rails.root.join('lib')
 
+    # Configure audit database
+    config.audit_database = ENV['DATABASE_URL']
```

Comment:
I don't understand this.. please explain the logic behind

Reply:
For OSS the config.audit_database is same as DATABASE_URL.
follower and master appliance should override this configuration